tasks/Module 3/task_2.py

```diff
@@ -0,0 +1,32 @@
+"""My split function."""
+
+
+def my_split(source_string, separator=None):
+    """My implementation of split() function."""
+    if separator is None:
+        separator = ' '
+    result = []
+    if len(source_string) < len(separator):
+        result.append(source_string)
```

Comment:
You can just return `[source_string]` here. But please also add a comment like `if the separator is longer than a string, there's no chance we can split the string in any way` or something similar.